doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

```diff
@@ -109,22 +108,12 @@ No application input is required to handle the requests.
 Bluetooth authentication
 ========================
 
-The Fast Pair service overrides the set of Zephyr's Bluetooth authentication callbacks (:c:struct:`bt_conn_auth_cb`) while handling Key-Based Pairing request.
-The callbacks are updated to report proper device capabilities for Bluetooth LE pairing request/response packets.
-Overriding callbacks allows the GFPS to take over Bluetooth authentication during `Fast Pair Procedure`_ and perform all of the required operations without interacting with the application.
-After the procedure is finished, the previously used callbacks are assigned back.
+The Fast Pair service uses :c:func:`bt_conn_auth_cb_overlay` to overlay set of Zephyr's Bluetooth authentication callbacks (:c:struct:`bt_conn_auth_cb`) used for a connection if the connected Bluetooth Central uses Fast Pair pairing flow.
```

Comment:
```suggestion
The Zephyr's Bluetooth authentication callbacks (:c:struct:`bt_conn_auth_cb`) are used for a connection if the connected Bluetooth Central uses Fast Pair pairing flow.
Fast Pair service uses :c:func:`bt_conn_auth_cb_overlay` to overlay a set of Zephyr's Bluetooth authentication callbacks (:c:struct:`bt_conn_auth_cb`).
```

Reply:
That's only partially true. Let me rephrase the complex sentence in another way.

Reply:
I wonder if changelog entry is necessary here. Fast Pair is still in experimental state. This change does not affect library API too (the underlying implementation has changed, but the code still performs the same task).